sdk/src/main/java/com/oursky/authgear/latte/WebViewClient.kt

```diff
@@ -50,6 +50,12 @@ internal class WebViewClient(
         error: WebResourceError?
     ) {
         super.onReceivedError(view, request, error)
+        // For unknown reason, when we attempt to inspect the webview with chrome,
+        // the webview will receive this error.
+        // But this is not a critical error so we ignore it here.
+        if (error != null && error.errorCode == -1 && error.description == "net::ERR_CACHE_MISS") {
```

Comment:
Use constant for error code?
https://developer.android.com/reference/android/webkit/WebViewClient#ERROR_UNKNOWN

Reply:
Updated.